rs/ledger_suite/icrc1/ledger/tests/tests.rs

```diff
@@ -540,63 +568,6 @@ fn balance_of(env: &StateMachine, ledger_id: CanisterId, account: Account) -> u6
     Decode!(&res, Nat).unwrap().0.to_u64().unwrap()
 }
 
-#[cfg_attr(feature = "u256-tokens", ignore)]
-#[test]
-fn test_upgrade_from_first_version() {
-    let env = StateMachine::new();
-
-    let ledger_wasm_first_version =
-        std::fs::read(std::env::var("IC_ICRC1_LEDGER_FIRST_VERSION_WASM_PATH").unwrap()).unwrap();
```

Comment:
What is this first version? Should it be dropped?

Reply:
Removed! cc @mbjorkqvist